USAGE_DATA.md

```diff
@@ -0,0 +1,17 @@
+## [Tekton Pipelines by Red Hat](https://github.com/redhat-developer/intellij-tekton)
+
+### Usage Data
+
+* when plugin is started
+* when an action is executed
+    * action name
+    * action duration time
+    * action error message (in case of exception)
+    * action specific data if applicable (see details below)
+* when plugin is shut down
+
+### Actions operating on tekton resources
+* resource kind
+* resource schema version
+* creation/update of a resource
+* namespace supports triggers
```

Comment:
Not quite clear to me, what does it mean exactly

Reply:
Reporting trigger support was removed. The entry in usage data should be removed.

Reply:
resolving this conversation. If there's more to it, feel free to reopen it.